src/librustc_mir/transform/instrument_coverage.rs

```diff
@@ -57,6 +57,7 @@ pub fn instrument_coverage<'tcx>(tcx: TyCtxt<'tcx>, body: &mut Body<'tcx>) {
             destination: Some((Place::from(temp), new_block)),
             cleanup: None,
             from_hir_call: false,
+            fn_span: span,
```

Comment:
Fixed new merge error.

Reply:
Tested locally, including the mir-opt test. This should fix the failure in the rollup log.

@Dylan-DPC 
@tmandry